compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -747,6 +757,18 @@ getEntry fp dar =
     maybe (fail $ "Package does not contain " <> fp) pure $
     findEntryByPath fp dar
 
+-- | Parse a manifest field.
+getManifestField :: Entry -> String -> IO String
+getManifestField manifest field =
+    mbErr ("Missing field in META-INF/MANIFEST.MD: " ++ field) $
+    headMay
+        [ value
+        | l <-
+              lines $
```

Comment:
Can we share this logic with the DAR reader? At the very least it would be good to add a comment on why we replace `\n ` as it took me a while to remember that this is for multiline fields.

Reply:
I moved the function to the dar reader. I think the dar reader itself might need some rewriting.